contracts/libraries/ClusterLib.sol

```diff
@@ -22,8 +22,11 @@ library ClusterLib {
         ISSVNetworkCore.Cluster memory cluster,
         uint64 burnRate,
         uint64 networkFee,
-        uint64 minimumBlocksBeforeLiquidation
+        uint64 minimumBlocksBeforeLiquidation,
+        uint64 minimumLiquidationCollateral
     ) internal pure returns (bool) {
+        if (cluster.balance < minimumLiquidationCollateral.expand()) return true;
```

Comment:
this is mainly why i point out the shrink
`minimumLiquidationCollateral.expand()`

Reply:
yes, so `minimumLiquidationCollateral` should be expanded to be comparable to `cluster.balance` that is a _expanded_ value (uint256)